src/main/java/com/onarandombox/MultiverseCore/display/ContentDisplay.java

```diff
@@ -1,268 +1,68 @@
-         * @param replacements  String formatting replacements.
-         * @return The builder.
-         */
-        @NotNull
-        public Builder<T> emptyMessage(@NotNull String emptyMessage, Object...replacements) {
-            this.display.emptyMessage = String.format(emptyMessage, replacements);
-            return this;
-        }
-
-        /**
-         * Sets the display handler that does the formatting and sending of content. <b>Required.</b>
-         *
-         * @param displayHandler    The display handler for the given content type.
-         * @return The builder.
-         */
-        @NotNull
-        public Builder<T> displayHandler(@NotNull DisplayHandler<T> displayHandler) {
-            this.display.displayHandler = displayHandler;
-            return this;
-        }
-
-        /**
-         * Sets the color tool used to make messages more colourful.
-         *
-         * @param colorTool The color tool to use.
-         * @return The builder.
-         */
-        @NotNull
-        public Builder<T> colorTool(@NotNull ColorTool colorTool) {
-            this.display.colorTool = colorTool;
-            return this;
-        }
-
-        /**
-         * Sets content filter used to match specific content to be displayed.
-         *
-         * @param filter    The filter to use.
-         * @return The builder.
-         */
-        @NotNull
-        public Builder<T> filter(@NotNull ContentFilter filter) {
-            this.display.filter = filter;
-            return this;
-        }
-
-        /**
-         * Sets other specific settings that may be used by the {@link DisplayHandler}.
-         *
-         * @param setting   The settings option.
-         * @param value     The value to set.
-         * @param <S>       The type of setting.
-         * @return The builder.
-         */
-        @NotNull
-        public <S> Builder<T> setting(@NotNull DisplaySetting<S> setting, S value) {
-            this.display.settingsMap.put(setting, value);
-            return this;
-        }
-
-        /**
-         * Validates and build the content display.
-         *
-         * @return The content display.
-         */
-        @NotNull
-        public ContentDisplay<T> build() {
-            Objects.requireNonNull(this.display.displayHandler);
-            return this.display;
-        }
-
-        /**
-         * Build and show the content to the sender.
-         *
-         * @param sender The CommandSender to show the display to.
-         */
-        public void show(CommandSender sender) {
-            this.build().show(sender);
-        }
+    public void send(@NotNull CommandSender sender) {
+        Objects.requireNonNull(sendHandler, "No send handler set for content display");
+        List<String> parsedContent = new ArrayList<>();
+        contentParsers.forEach(parser -> parser.parse(sender, parsedContent));
+        sendHandler.send(sender, parsedContent);
```

Comment:
Oh yeah, I think this makes a lot more sense. The builder pattern was a bit awkward for this.